Fix Terms, Term, MatchAll, MatchNone public docs (#9595)
opensearch-project/documentation-website: _query-dsl/term/terms.md

`id` | String | The document ID of the document from which to fetch field values. Required.
`path` | String | The name of the field from which to fetch field values. Specify nested fields using dot path notation. Required.
`routing` | String | Custom routing value of the document from which to fetch field values. Optional. Required if a custom routing value was provided when the document was indexed.

> `store` | Bool | Do the lookup on the stored field, instead of _source. Optional.

Reviewer: ```suggestion
`store` | Boolean | Whether to perform the lookup on the stored field instead of `_source`. Optional.
```
